connectors/salesforce/src/main/resources/cards/text.properties

```diff
@@ -25,10 +25,32 @@ opportunity.update.nextstep.label=Update Next Steps
 opportunity.update.closedate=Update Close Date
 opportunity.update.closedate.label=Update Close Date
 
-opportunity.update.nextstep=Update Next Steps
-opportunity.update.nextstep.label=Update Next Steps
+opportunity.se.header.title={0}
+opportunity.se.header.subtitle=Add activity
```

Comment:
You've specified this in the text.properties, however, you are hard coding "Add activity" in the java code.  It's easy to see by looking at your _xx test's json.  Anything not UPPERCASE should be the json data coming back from the salesforce call.  If it is our button labels, or field labels, titles, etc., then that means you either aren't defining it in the _xx.properties, OR you've made a mistake in the java code and aren't using your properties.

Reply:
Removed the unused properties. Added the missed ones in ```text_xx.properties```.